src/read.c

```diff
@@ -3599,7 +3615,11 @@ static avifResult avifCodecCreateInternal(avifCodecChoice choice, const avifTile
 
 static avifBool avifTilesCanBeDecodedWithSameCodecInstance(avifDecoderData * data)
 {
-    if (data->color.tileCount == 1 && (data->alpha.tileCount == 1 || data->gainMap.tileCount == 1)) {
+    avifBool hasSingleTileAlphaOrGainMap = data->alpha.tileCount == 1;
```

Comment:
Nit/optional: Another nonconstructive criticism: this varibale name is very long, but I don't have a good suggestion.

Reply:
I see what you mean but I think it's not _that_ long (it's only 2 chars longer than 'canUseSingleCodecInstance' a bit lower down) and I'd rather keep it descriptive.

Reply:
If the gain map is also an auxiliary image, then a good, slightly shorter name is `hasSingleTileAuxImage`.

The current name is also fine.

Reply:
No the gain map is not an auxiliary image, it's a derived item with a hidden input...